src/includes/enriching-events/scopes/scope-synchronization/android.mdx

```diff
@@ -0,0 +1,9 @@
+### Scope Synchronization
+
+If you want to set context data in the Java layer, then receive that context data when a native (C/C++) crash happens, you need to synchronize the Java `Scope` with the native `Scope`. This feature was introduced in version `3.0.0` of the Android SDK and requires an opt-in:
```

Comment:
Since this change isn't done yet, I'd leave it as it is and work with the migration docs once https://github.com/getsentry/sentry-java/issues/1914 lands.